visualizers/trace_visualizer/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "wheel"]
```

Comment:
I wonder if we should also list pip>24.0.0 here

Reply:
I tried it. But it is not complaining when I build it with a wrong pip version. I think these build system dependencies exist on a different level.

Reply:
nvm. I got it. It works.